test/setup/main.tf

```diff
@@ -16,7 +16,7 @@
 
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 3.0"
+  version = "~> 9.0"
 
   name              = "ci-bigquery"
   random_project_id = "true"
```

Comment:
nit: adding `skip_gcloud_download = true` can save us some minutes in CI :)

Reply:
looking forward to the 10.0 release... I already removed gcloud parameter from my head... hehe
thank you!